Sources/TokamakDOM/App.swift

```diff
@@ -44,12 +43,11 @@ extension App {
   /// The body is styled with `margin: 0;` to match the `SwiftUI` layout
   /// system as closely as possible
   ///
-  public static func _launch(_ app: Self,
-                             _ rootEnvironment: EnvironmentValues) {
-    let document = JSObjectRef.global.document.object!
-    let body = document.body.object!
-    let head = document.head.object!
-    body.style = "margin: 0;"
+  public static func _launch(_ app: Self, _ rootEnvironment: EnvironmentValues) {
+    let body = TokamakDOM.body
+    if body.style == .undefined {
+      body.style = "margin: 0;"
+    }
```

Comment:
@carson-katri what do you think about this check?

Reply:
That should do it 👍